tests/std/tests/P0355R7_calendars_and_time_zones_time_zones/timezone_data.h

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <chrono>
+#include <string>
+#include <string_view>
+
+using namespace std;
+using namespace std::chrono;
+
+class Transition {
+public:
+    Transition(
+        string_view name, seconds offset, minutes save, string_view abbrev, sys_seconds _begin_, sys_seconds _end_)
+        : _name(name), _offset(offset), _save(save), _abbrev(abbrev), _begin(_begin_), _end(_end_) {}
+
+    constexpr string_view name() const {
+        return _name;
+    }
+
+    constexpr seconds offset() const {
+        return _offset;
+    }
+
+    constexpr minutes save() const {
+        return _save;
+    }
+
+    constexpr string_view abbrev() const {
+        return _abbrev;
+    }
+
+    constexpr bool is_daylight() const {
+        return _save != minutes{0};
+    }
+
+    template <class _Duration = seconds>
+    constexpr sys_time<_Duration> begin() const {
+        return sys_time<_Duration>{duration_cast<_Duration>(_begin.time_since_epoch())};
+    }
+
+    template <class _Duration = seconds>
+    constexpr sys_time<_Duration> end() const {
+        return sys_time<_Duration>{duration_cast<_Duration>(_end.time_since_epoch())};
+    }
+
+    template <class _Duration = seconds>
+    constexpr local_time<_Duration> local_begin() const {
+        return local_time<_Duration>{duration_cast<_Duration>(_begin.time_since_epoch() + _offset)};
+    }
+
+    template <class _Duration = seconds>
+    constexpr local_time<_Duration> local_end() const {
+        return local_time<_Duration>{duration_cast<_Duration>(_end.time_since_epoch() + _offset)};
+    }
+
+private:
+    string _name;
+    seconds _offset;
+    minutes _save;
+    string _abbrev;
+    sys_seconds _begin;
+    sys_seconds _end;
+};
+
+// start of ambiguous/nonexistent zone between transitions
+template <class _Duration = seconds>
+constexpr local_time<_Duration> get_danger_begin(const Transition& first, const Transition& second) {
+    assert(first.end<_Duration>() == second.begin<_Duration>());
+    return first.local_end<_Duration>() - first.save();
+}
+
+// end of ambiguous/nonexistent zone between transitions
+template <class _Duration = seconds>
+constexpr local_time<_Duration> get_danger_end(const Transition& first, const Transition& second) {
+    assert(first.end<_Duration>() == second.begin<_Duration>());
+    return second.local_begin<_Duration>() + first.save();
+}
+
+// Sydney
+// Standard time (AEST : UTC+10) -1 @ 3am
+// Daylight time (AEDT : UTC+11) +1 @ 2am
+namespace Sydney {
+    static constexpr std::string_view Tz_name{"Australia/Sydney"sv};

```

Comment:
It doesn't really matter, but as namespace-scope constants in a header, these should ideally be `inline constexpr`, which fuses all definitions (even in separate translation units) together.